src/memprof_prejs.js

```diff
@@ -0,0 +1,8 @@
+// Will contain source of visualizations.
```

Comment:
This file seems to be used only for the test `test_memprof_compiler_flag`, so let's move this to under `tests/` and not `--pre-js` it by default in all builds. Instead that test can `--pre-js` it by itself?